src/errors/test-run/utils.js

```diff
@@ -0,0 +1,74 @@
+import dedent from 'dedent';
+import { escape as escapeHtml, repeat } from 'lodash';
+import TEST_RUN_PHASE from '../../test-run/phase';
+
+const SUBTITLES = {
+    [TEST_RUN_PHASE.initial]:                 '',
+    [TEST_RUN_PHASE.inFixtureBeforeHook]:     '<span class="subtitle">Error in fixture.before hook</span>\n',
+    [TEST_RUN_PHASE.inFixtureBeforeEachHook]: '<span class="subtitle">Error in fixture.beforeEach hook</span>\n',
+    [TEST_RUN_PHASE.inTestBeforeHook]:        '<span class="subtitle">Error in test.before hook</span>\n',
+    [TEST_RUN_PHASE.inTest]:                  '',
+    [TEST_RUN_PHASE.inTestAfterHook]:         '<span class="subtitle">Error in test.after hook</span>\n',
+    [TEST_RUN_PHASE.inFixtureAfterEachHook]:  '<span class="subtitle">Error in fixture.afterEach hook</span>\n',
+    [TEST_RUN_PHASE.inFixtureAfterHook]:      '<span class="subtitle">Error in fixture.after hook</span>\n',
+    [TEST_RUN_PHASE.inRoleInitializer]:       '<span class="subtitle">Error in Role initializer</span>\n',
+    [TEST_RUN_PHASE.inBookmarkRestore]:       '<span class="subtitle">Error while restoring configuration after Role switch</span>\n'
```

Comment:
Should not we remove '<span class=.....' from these constants?

Reply:
nop, they are ok